.changeset/neat-apples-boil.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/anchor': minor
```

Comment:
i know other i18n changesets were patches but should they be minors since we're adding new props?

Reply:
Hmmm yeah, I buy that. Good spot. They can't be retro fitted, and honestly for these its not the biggest mistake in the world, but I'm down for this a button being a minor